### PR TITLE
Add location page 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, there is no location page.

**What is the new behavior (if this is a feature change)?**
- The location page has been added.
- Something more has been added but James Donovan Halliday told me not to tell anyone. 🤫

**Other information**:
- The version has been updated to v1.1.0.

### CODE REVIEW

**Review by LucaAllmaier — APPROVED** [review]
Looks good to merge.

**Review by sweggerat — APPROVED** [review]
looks great

File: src/components/atoms/Egg/index.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  MDBPopoverHeader,
+  MDBPopoverBody,
+  MDBAnimation,
+} from "mdbreact";
+
+class Egg extends React.Component {
+  state = {};
+
+  componentDidMount = () => {
+    const res = localStorage.getItem("egg_" + this.props.id);
```
Comment (Aichnerc) [inline src/components/atoms/Egg/index.jsx:19]: Add a new line after declaring a variable.

File: src/components/atoms/Egg/index.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+              </span>
+            )}
+          </MDBPopoverBody>
+        </div>
+      </MDBPopover>
+    );
+  }
+}
+
+export default Egg;
```
Comment (Aichnerc) [inline src/components/atoms/Egg/index.jsx:82]: Missing license.

**Review by Aichnerc — COMMENTED** [review]
There are still some issues.

**Review by Aichnerc — COMMENTED** [review]
Looking good now.

**Review by pinterid — APPROVED** [review]
Looks good to merge.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

File: src/components/atoms/Egg/index.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    } else {
+      this.setState({
+        found: false,
+      });
+    }
+  };
+
+  setEgg = () => {
+    if (!this.state.found) {
+      this.setState(
```
Comment (schettn) [inline src/components/atoms/Egg/index.jsx:34]: You should change this to: 
this.setState({
        found: true,
        set: true,
},

**Review by schettn — CHANGES_REQUESTED** [review]
Should be ok after one change.

File: src/components/atoms/Egg/index.jsx (hunk 3 above)
Comment (Aichnerc) [reply src/components/atoms/Egg/index.jsx:34]: Wrong. Prettier tells you otherwise.

**Review by simonprast — APPROVED** [review]
Very nice

**Review by schettn — APPROVED** [review]
LGTM! 🦜

**Pull request merged**